Scripts/script-EmailAskUser.yml

```diff
@@ -138,4 +147,8 @@ args:
   - "true"
   - "false"
   defaultValue: "false"
+- name: retries
+  description: Indicates how many times to try and create an entitlement in case of failure
+  defaultValue: 10
 scripttarget: 0
+releaseNotes: Support script to run in parallel with itself
```

Comment:
Maybe rephrase - "Added support for parallel execution of the script, with better error handling"